app/controllers/api/v1/participants_controller.rb

```diff
@@ -27,7 +36,18 @@ def participants_query
       end
 
       def participant_params
-        params.permit(:ecf_id, filter: %i[updated_since])
```

Comment:
could we keep the updated since here?

Reply:
It doesn't appear to be used? we pull in a concern which exposes the `updated_since`:

https://github.com/DFE-Digital/npq-registration/blob/main/app/controllers/api/v1/participants_controller.rb#L5
https://github.com/DFE-Digital/npq-registration/blob/main/app/controllers/api/concerns/filter_by_updated_since.rb#L8